src/dispatch/plugins/dispatch_slack/commands.py

```diff
@@ -161,50 +168,68 @@ def filter_tasks_by_assignee_and_creator(tasks: List[Task], by_assignee: str, by
     return filtered_tasks
 
 
-async def handle_slack_command(*, client, request, background_tasks):
-    """Handles slack command message."""
-    # We fetch conversation by channel id
-    channel_id = request.get("channel_id")
-
-    db_session = get_organization_from_channel_id(channel_id=channel_id)
-    # We get the name of command that was run
+async def handle_non_incident_conversation_commands(client, request, background_tasks):
+    """Handles all commands that do not have a specific incident conversation."""
     command = request.get("command")
+    channel_id = request.get("channel_id")
+    command_args = request.get("text", "").split(" ")
+    if len(command_args):
+        organization_slug = command_args[0]
+
+    # We get the list of public and private conversations the Slack bot is a member of
```

Comment:
Making it more clear what bot we're talking about
```suggestion
    # We get the list of public and private conversations the Dispatch bot is a member of
```